src/helpers/storage.js

```diff
@@ -219,3 +232,50 @@ export const getUsableDevicesManualPartitioning = ({ devices, selectedDisks }) =
 
     return usableDevices;
 };
+
+const reuseManualPartitioning = ({ devices, partitioning, selectedDisks }) => {
+    const usableDevices = getUsableDevicesManualPartitioning({ devices, selectedDisks });
+
+    // Disk devices are not allowed in the mount point assignment
+    const usedDevices = (partitioning?.requests?.map(r => r["device-spec"]) || []);
+    if (usedDevices.every(d => usableDevices.includes(d)) && usableDevices.every(d => usedDevices.includes(d))) {
```

Comment:
Maybe we could use checkIfArraysAreEqual here but it is unrelated to the PR.

Reply:
used